frontend/src/components/login/login-input/LoginInput.module.scss

```diff
@@ -19,6 +19,10 @@
     background-color: $neutral-light;
   }
 
+  .info {
+    box-shadow: 1px 1px 1px 1px black;
```

Comment:
Curious, where did you get this style from?

Reply:
I initially attempted to just add a regular `border` but doing so conflicts with the existing styling of the component. So my solution was to use `box-shadow` in order to create a "border" that can be added onto the existing style of the component.

Reply:
Got it, we can re-visit if needed in the future. Great job on making a call on what's best!